content/root/best-practice-guide.textile

```diff
@@ -19,28 +19,57 @@ h2(#ably-101). Ably 101
 
 Ably Realtime is a cloud-based platform that allows you to easily integrate realtime functionality into your applications. Just plug and play!
 
-h3(#pubsub). Publish and Subscribe (Pub/Sub)
+h3(#auth). Authentication
 
-You can publish data to named "channels":/realtime/channels within Ably's platform and the platform will make sure that all clients subscribed to these channels will receive the data in realtime.
+Ably supports two main methods of authentication: 
 
-h3(#notifications). Notifications
+1. "Basic authentication":/core-features/authentication#basic-authentication 
+2. "Token authentication":/core-features/authentication#token-authentication
 
-Ably offers the "Push Notifications":/general/push feature which allows you to send notifications to your clients even when they are not using your application at the time, or in other words, not connected to Ably. This can be implemented on both mobile devices and browsers. Ably also offers a smart variation known as "Smart Notifications":/general/smart-notifications using which you can make the notifications most effective based on your user's connection state, their active device or even their current context within your application.
+For basic authentication you provide the Ably API key directly when authenticating with Ably. However, basic authentication should not be used on clients (for example a browser client) where the API key would be exposed. In production systems you should always avoid using basic authentication on the client-side - instead use "token authentication":/core-features/authentication#token-authentication using an @authUrl@ or @authCallback@.
 
-h3(#presence). Presence
+**Important:** Do not use basic authentication on the client-side, as it exposes your Ably API key.
 
-With Ably's "Presence":/realtime/presence feature, your clients can attach to a presence channel and announce to the rest of the users that they are present on a channel. This is very useful, for example, in chat apps to indicate a member is online, or with IoT devices to indicate a device is online and connected. Clients can subscribe to realtime presence changes on channels, such as members entering or leaving, and respond to these events accordingly.
+Token authentication allows you to connect your auth server to Ably, using the Ably API key. Your clients can then receive token requests from the auth server, and then authenticate directly with Ably using a token request. These short-lived tokens provide a secure way to pass information over the internet and make sure that the Ably API key is never exposed to your client-side applications. However, the secure auth server can use the API key directly to authenticate with Ably using basic authentication, as on the secure server the API key will not be accessible to anyone other than autheticated personnel, such as systems administrators.
```

Comment:
"Your clients can then receive token requests...." This sounds a little bit confused, because a 'request' is typically something the client asks for, rather than receives. Maybe we could say something like: "Your auth server uses your Ably API key to authenticate with Ably and provides a `TokenRequest` object to your client, which your client then uses to request a token. Subsequent operations use that token to authenticate with Ably, ensuring that your API key is never exposed to your client."

Reply:
Nice.